deploy/crds/push_v1alpha1_iosvariant_cr.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: IOSVariant
+metadata:
+  name: example-iosvariant
+spec:
+  description: 'My super iOS variant'
+  certificate: 'somekeyinbase64=='
+  passphrase: '123456'
```

Comment:
Should this maybe be base64 encoded also? I'm not sure what characters are allowed to be in the passphrase, and/or if any of those allowed characters could cause trouble here.

Reply:
Yeah, that's right. I just typed something there as the operator is not reading the CR at the moment. I suggest we update it later.